python/lsst/ts/ofc/scripts/generate_normalization_weights.py

```diff
@@ -0,0 +1,193 @@
+# This file is part of ts_ofc.
+#
+# Developed for Vera Rubin Observatory.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+import argparse
+
+import numpy as np
+import yaml
+from lsst.ts.ofc import BendModeToForce, OFCData, SensitivityMatrix
```

Comment:
Consider using relative import, since you are importing modules from the library;

```
from .. import BendModeToForce, OFCData, SensitivityMatrix
```